code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -101,18 +95,40 @@ adjustFireLoss(-(maxHealth / 70 + 0.5 + (maxHealth / 70) * recovery_aura/2)*(m))
 		adjustFireLoss(fire_stacks + 3)
 
 /mob/living/carbon/Xenomorph/proc/handle_living_health_updates()
+	if(health >= maxHealth || hardcore) //no damage, don't bother
+		updatehealth()
```

Comment:
Why update health if nothing is changing?

Reply:
`updatehealth()` does 3 things:
* Update health by recalculating it with the brute and fire loss.
* Update the medhud.
* Update the stat, killing the xeno if under a certain health, making it unconscious under another, and conscious over it.

It could be made more efficient, but the idea is that a xeno could transition from a state of low health to max, or one that warrants HUD changes. Suggestions?